### DISCUSSION TITLE
UserControl 的DataContext在设计器中报错，怎么排查，启动正常

### DISCUSSION THREAD

**Discussion by baciZzz** [BYJRK/DotNet-Discussions#299] [WPF 开发]
![QQ截图20240607170426](https://github.com/BYJRK/DotNet-Discussions/assets/160447610/ddde312e-c50c-42db-bb96-d11bf15ddca1)

ViewModel中的构造函数是空的

![微信图片编辑_20240607170633](https://github.com/BYJRK/DotNet-Discussions/assets/160447610/41cabfb3-c3f4-4532-83fc-90c9d944cdf1)

**Comment by BYJRK** [comment]
一般是你的 ViewModel 的构造访问了为空的变量，包括构造中调用的其他方法
